core/src/main/scala/kafka/log/LogCleaner.scala

```diff
@@ -127,7 +128,7 @@ class LogCleaner(val config: CleanerConfig,
    *  Abort the cleaning of a particular partition, if it's in progress. This call blocks until the cleaning of
    *  the partition is aborted.
    */
-  def abortCleaning(topicAndPartition: TopicAndPartition) {
+  def abortCleaning(topicAndPartition: TopicPartition) {
```

Comment:
In other places, we renamed `topicAndPartition` to `topicPartition`. Guess we could do the same here for consistency? A few more of these below.